src/main/java/net/torocraft/minecoprocessors/processor/Processor.java

```diff
@@ -540,6 +569,30 @@ void processJnc() {
     }
   }
 
+  void processJg() {
+    if(!carry && !zero) {
+      processJmp();
+    }
+  }
+
+  void processJge() {
+    if((!carry && !zero) || (!carry && zero)) {
+      processJmp();
+    }
+  }
+
+  void processJl() {
+    if(carry && !zero) {
+      processJmp();
+    }
+  }
+
+  void processJle() {
```

Comment:
@Ciastex I am writing some tests cases to protect these new commands from accidental changes, and having some trouble figuring out what they do.  Can you explain them and how they are using the flags.  I am starting to think they shouldn't be added without adding a `signed` flag to the processor based on this: http://faydoc.tripod.com/cpu/jg.htm

Reply:
I misnamed the instructions. x86 JA/JB/JAE/JBE work based on ZF/CF and perform an unsigned comparison. I tried to figure out the simplest way of implementing the comparison of two numbers using flags that are already provided. I used the description below and based the jumps according to the CMP instruction modification: https://www.aldeid.com/wiki/X86-assembly/Instructions/cmp

Reply:
I am going keep them as is for now, and update them and many other instructions if or when a SF and OF (maybe even a PF) are added